chain/client/src/client.rs

```diff
@@ -732,12 +803,12 @@ impl ClientActor {
         let epoch_start_height =
             unwrap_or_return!(self.runtime_adapter.get_epoch_start_height(&prev_block_hash), ());
 
-        debug!(target: "client", "Check announce account for {}, epoch start height: {}, {:?}", block_producer.account_id, epoch_start_height, self.last_val_announce_height);
+        debug!(target: "client", "Check announce account for {}, epoch start height: {}, {:?}", block_producer.account_id, epoch_start_height, self.last_validator_announce_height);
 
         // TODO MOO XXX: only announce if will be a validator in the next epoch.
```

Comment:
@SkidanovAlex this is already handled before announcement, we only send in case we are becoming a validator in the next epoch.